src/postgres/tree/PostgresFunctionsTreeItem.ts

```diff
@@ -18,6 +20,8 @@ export class PostgresFunctionsTreeItem extends AzureParentTreeItem<ISubscription
     public readonly parent: PostgresDatabaseTreeItem;
     public clientConfig: ClientConfig;
 
+    private _functionsAndSchemas: {}; // Function name to list of schemas
```

Comment:
Is something like this what you had in mind @nturinski? It made sense to just have func names as the properties instead of `schemas: string[]`

Reply:
Well, what I had was {schemas: string[]} which was type defining an object where the key is the function name and the value would be the array of schemas that have that name.

Reply:
DON'T YOU THUMBS UP A COMMENT WHERE I ADMIT TO BE DUMB WILL

Reply:
😂😂 You know what I mean